docs/documentation/make-first-prototype/show-users-answers.md

```diff
@@ -2,7 +2,7 @@
 
 If you do research with real user data, you must [open a new incognito window](https://support.google.com/chrome/answer/95464) for each research session, and close all windows before the next session.
```

Comment:
I've tweaked these two paras for style and conciseness in the version below:

If you do research with real user data, you must clear a user's data before each session by either:

- closing all browser windows and [opening a new incognito window](https://support.google.com/chrome/answer/95464)
- selecting the **Clear data** link at the bottom of the prototype

Reply:
I'll update this whenever I get the chance